apps/hashdotai/integrations/integration-list.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "name": "Linear",
+    "logoUrl": "https://dummyimage.com/160x160/000/fff&text=Linear",
+    "summaryHtml": "Issues, users, organizations, projects, milestones, and much more",
+    "connectUrl": "/integrations/linear/connect",
+    "aboutParas": [
+      {
+        "header": "Realtime two-way sync",
+        "copyHtml": "Create, update or delete data in either Linear or HASH while keeping them in sync.",
+        "iconUrl": "https://dummyimage.com/160x160/ddd/000000&text=two-way"
+      },
+      {
+        "header": "Jointly maintained by Linear & HASH",
+        "copyHtml": "<div><strong>This integration is officially supported by both Linear and HASH.</strong></div><div>It is jointly maintained to ensure robust, reliable and realtime operation.</div>",
```

Comment:
Per [this Slack comment](https://hashintel.slack.com/archives/C02NW18QGUR/p1697544637942309?thread_ts=1697542510.374379&cid=C02NW18QGUR), we want to define the acceptable `aboutParas` somewhere (outside of any one integration definition).

And we want to auto-insert the name of any given integration where it currently just says "Linear", rather than hard-code as here.

Reply:
Apologies, I forgot about that. What are the acceptable `aboutParas` to start with? Just something for the existing two, i.e. 'two-way' and 'jointly managed' or something like that?

Reply:
Sorry missed your reply here. Let's just go with these exact existing two as you suggest here for now. We'll add more as we encounter the need.

Reply:
Updated in https://github.com/hashintel/hash/pull/3415/commits/2f066b19ce9b48c7799bfcc659181439ebe13f44